editor/src/messages/input_preprocessor/input_preprocessor_message.rs

```diff
@@ -14,5 +15,6 @@ pub enum InputPreprocessorMessage {
 	PointerDown { editor_mouse_state: EditorMouseState, modifier_keys: ModifierKeys },
 	PointerMove { editor_mouse_state: EditorMouseState, modifier_keys: ModifierKeys },
 	PointerUp { editor_mouse_state: EditorMouseState, modifier_keys: ModifierKeys },
+	TimestampAdvance { timestamp: Duration },
```

Comment:
I think the purpose of the message would be clearer if it was named something like `FrameTimeAdvance` or `FrameTimeUpdate` to make it clearer what is meant by a timestamp. (Remember to also update `DEBUG_MESSAGE_BLOCK_LIST`— and if we could move that towards using the discriminant so it could be type-checked, that would be awesome.)